bin/postman-test/environment/local_edgex.postman_environment.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Can we both name this file and set the "name" property so that the default name is appropriate when imported into Postman? "support-logging.json" perhaps?

Reply:
yes , It's ok .